tests/behat/features/install/content/BasicPage.feature

```diff
@@ -52,7 +52,7 @@ Feature: Install State Basic Page
     Given I am logged in as a user with the "Contributor" role
     Then I am on "/node/add/hs_basic_page"
     And I should see 1 "#edit-field-hs-page-hero-wrapper" elements
-    And I should see 7 "#edit-field-hs-page-components-add-more input" elements
+    And I should see 8 "#edit-field-hs-page-components-add-more input" elements
```

Comment:
This seems like it will be a pain to maintain. Is the count really helpful or could it be removed?

Reply:
I kinda prefer it to make sure all the paragraphs are available in the field. its rare when we will add another paragraph type